### ISSUE TITLE
Advancement Not Showing

### ISSUE THREAD

**Issue by XtcCreeperALT** [DarwinBaker/AATool#74]
For the 1.20 tracker, the advancement "What a Deal" which requires you to trade with a Villager does not appear. It appears in 1.19, though.

![image](https://github.com/DarwinBaker/AATool/assets/138401306/7ff4040f-e047-46be-b44f-f539e52fabbf)
I can't seem to find any more though.

**Comment by DarwinBaker** [comment]
"What a Deal" was hidden for space/layout reasons. Since you get it when you do "Star Trader" anyway I figured it wouldn't be much of a problem to omit it from here since it's basically redundant. That being said I can see how it could cause confusion in certain scenarios. I combined "A Throwaway Joke" and "Very Very Frightening" into a single objective for the optimized layout, so maybe I'll do that here as well to free up an additional space to bring back "What a Deal"
